File: ChatApp.Server/Services/ViewModels/Recipients/RecipientViewModel.cs
namespace ChatApp.Server.Services.ViewModels.Recipients
{
    public class RecipientViewModel
    {
        public int Id { get; set; }
        public int RecipientTypeId { get; set; }
        public int? RecipientUserId { get; set; }
        public string? RecipientUsername { get; set; }
        public int? RecipientGroupId { get; set; }
        public string? RecipientGroupName { get; set; }
        public DateTime CreatedAt { get; set; }
        public DateTime ModifiedAt { get; set; }
    }
}

File: chatapp.client/src/app/models/recipient-view-model.ts
export class RecipientViewModel {
  id: number = 0;
  recipientTypeId: number = 0;
  recipientUserId: number | null = null;
  recipientUsername: string | null = null;
  recipientGroupId: number | null = null;
  recipientGroupName: string | null = null;
  createdAt: string = '';
  modifiedAt: string = '';
}
